pyuvsim/tests/test_analyticbeam.py

```diff
@@ -202,16 +189,17 @@ def test_gaussbeam_values():
 
     engine = pyuvsim.UVEngine(task)
     engine.apply_beam()
-    altitudes = task.sources.alt_az[0]   # In radians.
-    coherencies = np.real(engine.apparent_coherency[0, 0] + engine.apparent_coherency[1, 1]).astype(float)  # All four components should be identical
+    altitudes = task.sources.alt_az[0]  # In radians.
+    coherencies = np.real(engine.apparent_coherency[0, 0] + engine.apparent_coherency[1, 1]).astype(
+        float)  # All four components should be identical
```

Comment:
I think this could be an extension of the line-breaking comment too, but it is a test so not necessarily mission critical.

Reply:
Fixed in b7e89d70fab47c00273dbfa95fa5f5c82f3af8b0